unirest/src/main/java/kong/unirest/apache/ApacheResponse.java

```diff
@@ -81,10 +81,12 @@ public InputStream getContent() {
         }
         try {
             InputStream is = getContent();
-            if (isGzipped(getEncoding())) {
-                is = new GZIPInputStream(getContent());
+            byte[] bytes = getBytes(is);
```

Comment:
First of all, thanks for the PR 🥳 ! I only have one issue I would like to address. This results in consuming the entire InputStream which could be large and chunked. This would be a problem if someone was using Unriest to download a large payload. 

What would be better would be to leverage ```available()``` on the  original InputStream. this method:
> Returns an estimate of the number of bytes that can be read (or skipped  over) from this input stream without blocking, which may be 0, or 0 when end of stream is detected.

Reply:
Made changes as suggested. Thanks